Makefile.dep

```diff
@@ -429,3 +429,8 @@ endif
 ifneq (,$(filter ethos,$(USEMODULE)))
     USEMODULE += netdev2_eth
 endif
+
+ifneq (,$(filter cc110x,$(USEMODULE)))
+    USEMODULE += xtimer random
+    DISABLE_MODULE += gnrc_sixlowpan_iphc
```

Comment:
It didn't work, probably because of misformed link-local addresses. Have to verify if that is still the case.